src/js/randomizer/tokens/token_canvas.ts

```diff
@@ -19,7 +19,6 @@ function splitLinesCircle(
     const lineWidth = circleWidthAt(radius, radius - y) * widthFraction;
     // the current line is text.slice(0, i);
     let i = -1;
-    // eslint-disable-next-line no-constant-condition
```

Comment:
> Unused eslint-disable directive (no problems were reported from 'no-constant-condition').

Reply:
That default is sensible and I don't think we need to make it explicit in the config